Automattic/simplenote-macos
# Tracks: Opt Out

### Details:
This PR implements Tracks Opt Out, via Simperium Sync, and a super simple Privacy UI (based on the About UI).

@roundhill i'd be more than happy to twean the Privacy UI, as needed. Screenshots below (Dark / Light mode in Mojave).

Thanks in advance!!!

Closes #238

### Scenario: Sync'ing
1. Launch the latest iOS Version (with the Privacy Settings)
2. Launch Simplenote macOS and open the Privacy Settings
3. Toggle iOS's Privacy settings
4. Verify macOS nearly instantaneously refreshes, and displays the new setting

### Scenario: Learn More
1. Launch Simplenote Mac
2. Open the Privacy Settings
3. Click over the `Help us improve (...)` Label (or right image)
4. Verify Safari shows up with our Privacy Policy

### Scenario: Opt Out
1. Check out this commit: [2aa1b8e](
https://github.com/Automattic/simplenote-macos/pull/242/commits/2aa1b8e574e7ae2bb56c5ef0fca4b663b2e75ca5)
2. Enable the Tracks Opt Out
3. Trigger anything that might result in an event (ie. open a note)
4. Verify the following shows up in the console: `## Disregarding Event (...)`
5. Enable Tracks
6. Trigger something that might result in an event (ie. open a note)
7. Verify the following shows up in the console: `## Sending Event (...)`


---

<img width="1176" alt="screen shot 2018-09-25 at 11 25 15 am" src="https://user-images.githubusercontent.com/1195260/46020861-b7cbcb00-c0b5-11e8-8df1-bb290b2eb535.png">

<img width="1176" alt="screen shot 2018-09-25 at 11 25 35 am" src="https://user-images.githubusercontent.com/1195260/46020888-c31ef680-c0b5-11e8-8df3-b311d5fe9940.png">
